resources/l10n/ja/LC_MESSAGES/loot.po

```diff
@@ -445,142 +445,142 @@ msgid ""
+#: src/gui/qt/main_window.cpp:1614
 msgid ""
 "The plugin metadata editor is already open, first close the editor before "
 "attempting to edit another plugin's metadata."
 msgstr ""
 "すでに画面が開かれています。別のプラグインのメタデータを編集するには、現在の"
 "メタデータ編集画面を閉じてください。"
 
-#: src/gui/qt/main_window.cpp:1665
+#: src/gui/qt/main_window.cpp:1672
 msgid "The metadata for \"{0}\" has been copied to the clipboard."
 msgstr "\"{0}\" のメタデータをクリップボードにコピーしました。"
 
-#: src/gui/qt/main_window.cpp:1683
+#: src/gui/qt/main_window.cpp:1690
 msgid "The plugin name \"{0}\" has been copied to the clipboard."
 msgstr "\"{0}\" のプラグイン名をクリップボードにコピーしました。"
 
-#: src/gui/qt/main_window.cpp:1702
+#: src/gui/qt/main_window.cpp:1709
 msgid "The card content for \"{0}\" has been copied to the clipboard."
 msgstr "\"{0}\" のカード内容をクリップボードにコピーしました。"
 
-#: src/gui/qt/main_window.cpp:1717
+#: src/gui/qt/main_window.cpp:1724
 msgid ""
 "Are you sure you want to clear all existing user-added metadata from \"{0}\"?"
 msgstr "\"{0}\" のユーザーメタデータを消去しますか？"
 
-#: src/gui/qt/main_window.cpp:1757
+#: src/gui/qt/main_window.cpp:1764
 msgid "The user-added metadata for \"{0}\" has been cleared."
 msgstr "\"{0}\" のユーザーメタデータを消去しました。"
 
-#: src/gui/qt/main_window.cpp:1808
+#: src/gui/qt/main_window.cpp:1815
 msgid "Version {0} (build {1})"
 msgstr "バージョン {0} (ビルド {1})"
 
-#: src/gui/qt/main_window.cpp:1814
+#: src/gui/qt/main_window.cpp:1821
 msgid ""
 "Load order optimisation for Morrowind, Oblivion, Nehrim, Skyrim, Enderal, "
 "Skyrim Special Edition, Enderal Special Edition, Skyrim VR, Fallout 3, "
 "Fallout: New Vegas, Fallout 4 and Fallout 4 VR."
 msgstr ""
-"このソフトはMorrowind、Oblivion、Nehrim、Skyrim、Enderal、Skyrim Special "
+"このソフトは Morrowind、Oblivion、Nehrim、Skyrim、Enderal、Skyrim Special "
 "Edition、Enderal Special Edition、Skyrim VR、Fallout 3、Fallout: New Vegas、"
-"Fallout 4、Fallout 4 VRのロードオーダー（ロード順）を最適化します。"
+"Fallout 4、Fallout 4 VR のロードオーダー（ロード順）を最適化します。"
 
-#: src/gui/qt/main_window.cpp:1831
+#: src/gui/qt/main_window.cpp:1838
 msgid "About LOOT"
-msgstr "LOOTについて"
+msgstr "LOOT について"
 
-#: src/gui/qt/main_window.cpp:2154
+#: src/gui/qt/main_window.cpp:2142
 msgid "Identifying conflicting plugins..."
 msgstr "競合するプラグインを特定中…"
 
-#: src/gui/qt/main_window.cpp:2278
+#: src/gui/qt/main_window.cpp:2285
 msgid ""
 "Auto-sort has been cancelled as there is at least one error message "
 "displayed."
 msgstr ""
-"1つ以上のエラーメッセージが表示されているため自動ソートは中止されました。"
+"1 つ以上のエラーメッセージが表示されているため自動ソートは中止されました。"
 
-#: src/gui/qt/main_window.cpp:2329
+#: src/gui/qt/main_window.cpp:2336
 msgid "No masterlist update was necessary."
 msgstr "現在マスターリストの更新は必要ありません。"
 
-#: src/gui/qt/main_window.cpp:2342
+#: src/gui/qt/main_window.cpp:2349
 msgid "Masterlist updated to revision {0}."
 msgstr "マスターリストがリビジョン {0} に更新されました。"
 
-#: src/gui/qt/main_window.cpp:2400
+#: src/gui/qt/main_window.cpp:2407
 msgid "A [new release]({0}) of LOOT is available."
-msgstr "[新しいバージョンのLOOT]({0})が利用できます。"
+msgstr "[新しいバージョンのLOOT] ({0}) が利用できます。"
```

Comment:
The space between the `]` & `(` should be removed as that's a Markdown hyperlink. Did you mean to put it before `LOOT` like in the rest of the PR?

```suggestion
msgstr "[新しいバージョンの LOOT]({0}) が利用できます。"
```

Reply:
Thank you for your review! As you pointed out, I was trying to insert the space in the wrong position. It should be fixed now.